sdk/textanalytics/ai-text-analytics/samples/javascript/README.md

```diff
@@ -61,6 +62,7 @@ Take a look at our [API Documentation][apiref] for more information about the AP
 
 [alternativedocumentinput]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/textanalytics/ai-text-analytics/samples/javascript/alternativeDocumentInput.js
 [analyzesentiment]: https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/textanalytics/ai-text-analytics/samples/javascript/analyzeSentiment.js
+[analyzesentimentwithopinionmining]: https://github.com/deyaaeldeen/azure-sdk-for-js/blob/opinion-mining/sdk/textanalytics/ai-text-analytics/samples/javascript/analyzeSentimentWithOpinionMining.js
```

Comment:
This should link to master. 
If your CI doesn't allow it until the file actually exist, I recommend creating an issue so you can address it after merging the PR

Reply:
[Done](https://github.com/Azure/azure-sdk-for-js/issues/10559).